pages/task/read/views/pil.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Fragment } from 'react';
```

Comment:
We might want to move the PIL component up to common because it will probably be useful on other pages?

Reply:
Leave it where it is for now. If and when it becomes useful elsewhere it can be moved.